ols/app/metrics/metrics.py

```diff
@@ -53,3 +59,14 @@ def get_metrics(auth: Any = Depends(auth_dependency)) -> Response:
         Response containing the latest metrics.
     """
     return Response(content=generate_latest(), media_type=CONTENT_TYPE_LATEST)
+
+
+def setup_model_metrics(config: config_model.Config) -> None:
+    """Perform setup of all metrics related to LLM model and provider."""
+    selected_provider.info({"name": config.ols_config.default_provider})
+    selected_model.info({"name": config.ols_config.default_model})
+
+    for _, provider in config.llm_config.providers.items():
+        provider_name = provider.name
+        for model_name, _ in provider.models.items():
+            model_enabled.labels(provider_name, model_name).inc()
```

Comment:
we should label by provider type, not name.  Name is chosen by the user, type has to be one of our known providers

Reply:
is `type` set up in all cases? (I thought it is optional). But easy change, lemme do it

Reply:
updated accordingly